src/react-components/src/components/01-atoms/Button/Button.stories.tsx

```diff
@@ -8,6 +8,38 @@ export default {
   component: Button,
 };
 
-export const buttonFilledIcon = () => <Button callback={action("clicked")} id="button" content="search" type="filled" iconPosition="left" iconName="search-small" iconDecorative={true}/>;
+export const buttonFilledIcon = () =>
+  <Button
+    callback={action("clicked")}
+    id="button"
+    content="search"
+    type="filled"
+    iconPosition="left"
+    iconName="search-small"
+    iconDecorative={true}
+  />
+;
 export const buttonOutline = () => <Button callback={action("clicked")} id="button" content="Hello World" type="outline" />;
 export const buttonWithSpan = () => <Button callback={action("clicked")} id="button" type="outline" content={<span>Style my span!</span>}/>;
+export const buttonIconChildrenString = () =>
+  <Button
+    callback={action("clicked")}
+    id="button"
+    type="filled"
+    iconPosition="right"
+    iconName="search-small"
+    iconDecorative={true}
+  >Search!</Button>
+;
```

Comment:
nit

Reply:
I'm not sure I know what this means.

Reply:
oh uh sorry for my weird shorthand -- I'm not sure which linter you're using?  It's weird to have the semicolon on a separate line here (and in line 21) 
(and you're missing the empty line at end of file)

Reply:
That's weird. The linter doesn't complain, does it for you?